.travis.yml

```diff
@@ -17,4 +17,6 @@ matrix:
       script:
         #- make website-test #commented until Terraform Provider Program Validation
         - make website-lint
-script: make test
+    - name: "Test"
+      script:
+        - make test
```

Comment:
This still shows no newline at the end of the file, it passed the linter but should probably fix it up.

Reply:
weird, I added newline, gotta check again

Reply:
I did not realize that is the travis, I'm not sure if at the end we should keep the .travis.yml if Github actions can do the same work and "faster"

Reply:
We'll probably remove it once we get github actions working well but let's keep it for now.